docs/word/citation-management.md

```diff
@@ -0,0 +1,68 @@
+---
+title: 'Sample: Manage citations through your Word add-in'
+description: Learn how to create an add-in that manages citations in a Word document.
+ms.date: 11/15/2023
+ms.topic: sample
+ms.localizationpriority: medium
+---
+
+# Sample: Manage citations in a Word document using your Word add-in
+
+Citation management is an important aspect of documents, particularly in academia and education. Each citation style has its own guidelines for how citations should be marked in a document as well as where and how the sources should be noted. Such styles include [APA](http://apastyle.apa.org/) and [MLA](https://www.mla.org/MLA-Style).
+
+This article features a sample add-in that manages citations in a Word document. The add-in displays the references loaded from the user's .bib file that the user selects to cite in their document.
+
+![Screen showing citation management add-in task pane.](../images/word-citation-management.png)
```

Comment:
I think we're supposed to avoid saying "screen" or "screenshot".

```suggestion
![The citation management add-in task pane.](../images/word-citation-management.png)
```